### PR TITLE
Enhancement: Show unread marker

### PR DESCRIPTION
**Pull request by SanjuSudheerM**
#3343 

## Description

Show unread marker on the widget button

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Set up an inbox and send messages from the agent side to the customer. The unread messages will be shown on top of the chat bubble. on closing the unread message popup, the notification dot will appear on the right top of the chat bubble as mentioned in the sample image attached to the ticket. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Screenshot 2021-11-15 at 4 30 08 PM

### CODE REVIEW

**Review by nithindavid — DISMISSED** [review]

**Comment by sojan-official** [comment]
let's also test the behavior of the indicator in case of the expanded bubble while QAing this.

**Comment by nithindavid** [comment]
@SanjuSudheerM The circle ci is failing because of a lint error? Can you fix that too?

**Comment by SanjuSudheerM** [comment]
@nithindavid fixed the lint error.

File: app/javascript/sdk/sdk.js [hunk 1]
```diff
@@ -37,6 +37,19 @@ export const SDK_CSS = `.woot-widget-holder {
   user-select: none;
 }
 
+.woot-widget-bubble.unread-notification::after {
+  content: '';
+  position: absolute;
+  width: 14px;
```
Comment (nithindavid) [inline app/javascript/sdk/sdk.js:43]:
```suggestion
  width: 12px;
```

File: app/javascript/sdk/sdk.js [hunk 2]
```diff
@@ -37,6 +37,19 @@ export const SDK_CSS = `.woot-widget-holder {
...
+  height: 14px;
```
Comment (nithindavid) [inline app/javascript/sdk/sdk.js:44]:
```suggestion
  height: 12px;
```

**Review by nithindavid — APPROVED** [review]
Clean PR! Thanks @SanjuSudheerM 🔥

**Pull request merged**